service/ocean/providers/aws/cluster.go

```diff
@@ -119,9 +119,38 @@ type ShutdownHours struct {
 }
 
 type Task struct {
-	IsEnabled      *bool   `json:"isEnabled,omitempty"`
-	Type           *string `json:"taskType,omitempty"`
-	CronExpression *string `json:"cronExpression,omitempty"`
+	IsEnabled      *bool          `json:"isEnabled,omitempty"`
+	Type           *string        `json:"taskType,omitempty"`
+	CronExpression *string        `json:"cronExpression,omitempty"`
+	Parameter      *Parameter     `json:"parameters,omitempty"`
+	AmiAutoUpdate  *AmiAutoUpdate `json:"amiAutoUpdate,omitempty"`
```

Comment:
please remove AmiAutoUpdate from here, as it is part of Parameter object

Reply:
Removed